scripts/hotfixes/kdump/dracut-lib.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```

Comment:
```suggestion
# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
```

Reply:
The original script has been around since 2021, so this should probably align to that.